src/main/java/com/uploadcare/api/Image.java

```diff
@@ -0,0 +1,118 @@
+package com.uploadcare.api;
+
+import com.uploadcare.data.FileData;
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
+    public enum Orientation {
```

Comment:
I don't think so, maybe later. I think it is useful for small programs where just request image form uploadcare, check size, format, orientation and send it to the front end.